src/library/playlistfeature.cpp

```diff
@@ -248,6 +248,10 @@ QString PlaylistFeature::getRootViewHtml() const {
                                             playlistsSummary4));
     html.append("</td></tr>");
     html.append(QString("<tr><td><a href=\"create\">%1</a>")
+    //Colorize links in lighter blue, instead of QT default dark blue.
+    //Links are still different from regular text, but readable on dark/light backgrounds.
+    //https://bugs.launchpad.net/mixxx/+bug/1744816
+    html.append(QString("<a style=\"color:#0850D0;\" href=\"create\">%1</a>")
```

Comment:
it works and looks like desired if the line is
```
html.append(QString("<tr><td><a style=\"color:#0850D0;\" href=\"create\">%1</a>")
                    .arg(createPlaylistLink));
```

Reply:
fixed with  83b515e

Reply:
> fixed with 83b515e
What is going on here, to which branch went this, and the other commits?
I can´t find them..

Reply:
Oh, sorry for being unclear. I made those commits to my fork of your 2.1 branch [deere-crate-link-fix-fixed](https://github.com/ronso0/mixxx/tree/deere-crate-link-fix-fixed) which you could just merge into yours. It contains:
- create crate/playlist: fix html link
- make skin settings icon a bit smaller
- settings icon: remove SVG attributes 'width' and 'height' so it resized properly in the skin
- use skin settings tooltip in Tango